templates/workspace_services/guacamole/user_resources/guacamole-azure-importreview-vm/template_schema.json

```diff
@@ -0,0 +1,38 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "https://github.com/microsoft/AzureTRE/templates/workspace_services/guacamole/user_resources/guacamole-azure-windowsvm/template_schema.json",
+    "type": "object",
+    "title": "Import review Virtual Machine",
+    "description": "Windows virtual machine for import review",
+    "required": [
+    ],
```

Comment:
Here, let's add a following block

```json
"authorizedRoles": [
    "AirlockManager"
],
```

This will mean that only Airlock Managers will be allowed to deploy this. 
(In fact, users whose only role is Airlock Manager won't be allowed to deploy that anyway - for this, we will need https://github.com/microsoft/AzureTRE/pull/2508 )